Phases/Assessments/README.md

```diff
@@ -0,0 +1,131 @@
+# Phase Assessments
+
+This document explains how phase assessments will work. To learn what each phase assessment will cover and how to prepare, see the phase-specific assessment pages:
+
+- [Phase 1](./Phase_1.md)
+- [Phase 2](./Phase_2.md)
+- [Phase 3](./Phase_3.md)
+
+Note that there is no assessment for phases 4 or 5.
```

Comment:
Maybe a simpler way of conveying:

"Note that there are no more assessments once you've reached phase 4."